src/buildblock/ProjDataInfo.cxx

```diff
@@ -453,7 +449,7 @@ ProjDataInfo::get_empty_related_viewgrams(const ViewSegmentNumbers& view_segment
     viewgrams.push_back(get_empty_viewgram(pairs[i].view_num(),
                                           pairs[i].segment_num(),
 										  make_num_tangential_poss_odd,
-										  timing_pos_num));
+										  pairs[i].tof_pos_num()));
```

Comment:
I think we should have a `get_empty_viewgrams(viewgram_indices)` new member, which would clean this up nicely.